action.yml

```diff
@@ -39,6 +39,9 @@ inputs:
   authPayload:
     description: 'The JSON payload to be sent to Vault when using a custom authentication method.'
     required: false
+  postPayload:
+    description: 'The JSON payload to be sent to Vault over a POST request, enabled POST instead of GET.'
+    required: false
```

Comment:
I think we should document this new input in the [README alongside the ones already documented there](https://github.com/Responsive-Software/vault-action/blob/provide-post/README.md?plain=1#L386).